client/signup/steps/design-picker/index.jsx

```diff
@@ -26,7 +36,54 @@ class DesignPickerStep extends Component {
 		showOnlyThemes: false,
 	};
 
+	state = {
+		selectedDesign: null,
+		// Only offering designs that are also available as themes. This means excluding
+		// designs where the `template` has a layout that's different from what the theme's
+		// default Headstart annotation provides.
+		availableDesigns: getAvailableDesigns().featured.filter(
+			( { features, template, theme } ) => theme === template && ! features.includes( 'anchorfm' )
+		),
+		scrollTop: 0,
+	};
+
+	componentDidUpdate( prevProps ) {
+		if ( prevProps.stepSectionName !== this.props.stepSectionName ) {
+			this.updateSelectedDesign();
+			this.updateScrollPosition();
+		}
+	}
+
+	updateScrollPosition() {
+		if ( this.props.stepSectionName ) {
+			this.setState( { scrollTop: document.scrollingElement.scrollTop } );
+			document.scrollingElement.scrollTop = 0;
```

Comment:
It looks to me like you've found a CSS solution for the height of this page as well, so it looks like the javascript scrolling fix is no longer neccessary.

Reply:
Maybe we should keep this if the height still makes vertical overflow in the certain device or certain window size?

Reply:
Okay, then I'll remove it